build/DependencyVersions.props

```diff
@@ -4,10 +4,12 @@
     <CLI_SharedFrameworkVersion>2.0.0</CLI_SharedFrameworkVersion>
     <CLI_MSBuild_Version>15.5.0-preview-000154-1054573</CLI_MSBuild_Version>
     <CLI_Roslyn_Version>2.6.0-beta1-62126-01</CLI_Roslyn_Version>
-    <CLI_Roslyn_Satellites_Version>2.6.0-pre-20171012-1</CLI_Roslyn_Satellites_Version>
     <CLI_DiaSymNative_Version>1.6.0-beta2-25304</CLI_DiaSymNative_Version>
     <CLI_FSharp_Version>4.2.0-rtm-170926-0</CLI_FSharp_Version>
-    <CLI_FSharp_Satellites_Version>4.4.1-pre-20171003-1</CLI_FSharp_Satellites_Version>
+
+    <CLI_Deps_Satellites_Build>pre-20171012-1</CLI_Deps_Satellites_Build>
+    <CLI_Roslyn_Satellites_Version>2.6.0-pre-$(CLI_Deps_Satellites_Build)</CLI_Roslyn_Satellites_Version>
```

Comment:
When do we have to update the front numbers? Do we ever have to?

Reply:
Right now, they match the assembly versions of roslyn and F# so once per release max. Honestly, I could have just used any 3 digits for package version and I regret not using the same version for both. This guy should go away in 15.6, but if the assembly versions ever change again, I'll change the package version to be something constant.